satpy/tests/test_writers.py

```diff
@@ -30,7 +30,7 @@
 import numpy as np
 import xarray as xr
 
-from satpy.writers import show, to_image
+from satpy.writers import show, to_image, compute_writer_results
```

Comment:
Could you move the satpy imports to be inside the tests so that import errors/failures on satpy's side of things don't cause all tests to not run?